### DISCUSSION TITLE
Unsure about remote and IP

### DISCUSSION THREAD

**Discussion by rezna11** [GerryDazoo/Slinger#524] [Q&A]
Hello. I've managed to stream from my Slingbox 350. But the remote which has been loaded into the Slingbox many years ago is not what I'd like to use. There's quite a lot of discussion about custom remotes here but I wasn't able to find the answer to a simple question: is it possible to change the remote on the Slingbox to control different device?

I also have a Slingbox Solo and Pro HD. But those seem to have a static IP configured and since those were offline for almost 10 years, I have no idea what IP range they were on. Am I assuming correctly that there's no way to change the network settings to DHCP now? Resetting the Slingboxes would brick them forever, correct?

Thanks

**Comment by sp222222** [comment]
you gotta read up in here ---- on your router that has the slingbox on it ----BLOCK OUTbound port 5678 or you will BRICK your SOLO and PRoHD......350 is fine. 

do NOT hard reset the 350. you CAN hard reset the  solo/proHD IF you cannot remember the admin pass

**Comment by rezna11** [comment]
Well I'm reading... but still really not sure how to set up the remote. I got ProHD and 350 streaming just fine by now. The Solo is probably dead (error 0x92340070). But no idea how to set up the remote control.

On ProHD I tried to setup the remote with Slingplayer 1.5. It worked from the setup assistant, but apparently never got saved into the Slingbox. So I wonder how do I save a remote definition (ideally a custom PL binary file) to the Slingbox, specifically ProHD and 350...

**Reply by GerryDazoo** [reply]
How do you know it wasn't saved to the Slingbox? I'm unaware of any issues with setting the video source on the Pro with SPD 1.5.

**Comment by rezna11** [comment]
Because the IR codes info was empty. But because you wrote that it should work, I tried several times again and finally it worked. Before I usually skipped the remote viewing setup. Now I went on with it and it worked and saved the remote. Or maybe it was something else. Not sure. But it's great. 

Just one issue left with the ProHD - the Slingbox logo keeps glowing all the time. Any way to get rid of it?

And how about the 350? Is there a way to set up the sources and remotes?

**Comment by rezna11** [comment]
But unfortunately the remote doesn't survive a restart...
After finishing the setup:
![image](https://user-images.githubusercontent.com/20435164/218107407-b1779b17-39f1-4d98-a460-e7b69975bf4f.png)

And after restarting the Slingbox:

![image](https://user-images.githubusercontent.com/20435164/218108498-96179198-bd54-4a4a-85ba-8d1dcde7dcfb.png)

**Reply by GerryDazoo** [reply]
On the ProHD video source zero is the tuner I think. You can't configure a remote on that source. Try configuring the Component input.

**Reply by rezna11** [reply]
Sure, the AV input in use is Component, the one I configured. 
![image](https://user-images.githubusercontent.com/20435164/218141593-348f9d0e-07ce-4d72-a9d0-9c233be018bd.png)


Here's a better screenshot:

![image](https://user-images.githubusercontent.com/20435164/218139157-61c75f3b-529a-417c-8634-a2ba078aa9dc.png)

But I'm not sure if the remote is stored properly because the setup always ends with an error. 

either this
![image](https://user-images.githubusercontent.com/20435164/218140989-33beadde-eecd-45fc-b7a1-fa3c22c70a2d.png)

or this if I skip the remote viewing setup (after several minutes of "Configuring Slingbox settings..." message

![image](https://user-images.githubusercontent.com/20435164/218142987-448c3da2-610e-409d-9e07-81f52b827a8f.png)


But in both cases the Slinger web remote works right after... until a Slingbox restart. Anyway as I said, the Sling logo keeps blinking and glowing all the time, suggesting something is definitely not right. Also the Slingplayer 1.5 always asks me to set up the Slingbox, so it's seen as not set up properly.

**Reply by GerryDazoo** [reply]
What ports do you have blocked on your firewall? you just need to block TCP port 5678 outgoing from your Slingbox. 
Also are you configuring a static IP for the ProHD?

**Reply by rezna11** [reply]
I tried with all internet traffic from/to the Slingbox blocked or only 5678 outgoing blocked but with same result. So let me just ask - should I be able to complete the Slingbox setup process, including setup of remote viewing, in the Slingplayer 1.5, without any errors at the end?

**Reply by GerryDazoo** [reply]
Yes. But I just noticed a UK reference in your screen shots. I think there is a special SPD binary you need to use. Please have another look at the ProHD un-bricking procedure.

**Comment by rezna11** [comment]
Thanks for helping out. I finally got it working with one of my other ProHDs. Also EU model. But the other EU model and one US model - no luck. Tried both EU and US Slingplayer 1.5 versions.

**Comment by rezna11** [comment]
OK, issues with ProHD resolved. I found this guide https://www.mediafire.com/file/ppw6ep09t99xhu0/Slingbox_ProHD_PC_Unbricking-Setup-Network-Guide.pdf/file and followed this way of setup, and it worked for the other ProHD too! 
Now if I just could fix the remote for the 350...
